app/containers/LoginHome/index.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { TextField } from 'material-ui';
+import styled from 'styled-components';
+import { ms } from '../../styles/helpers';
+import Button from '../../components/Button/';
+import Checkbox from '../../components/Checkbox/';
+import TermsModal from '../../components/TermsModal/';
+import { shell } from 'electron';
+import styles from './styles.css';
+
+const SectionContainer = styled.div`
+  display: flex;
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const TermsAndPolicySection = styled.div`
+  display: flex;
+  width: 80%;
+  padding: ${ms(2)} 0 ${ms(4)} 0;
+  border-top-width: 1px;
+  border-top-color: ${ ({ theme: { colors } }) => colors.gray3 };
+  border-top-style: solid;
+  justify-content: center;
+  align-items: center;
+`;
+
+const Strong = styled.span`
+  color: ${ ({ theme: { colors } }) => colors.accent };
+`;
+
+const Link = styled(Strong)`
+  cursor: pointer;
+`;
+
+const Description = styled.span`
+  color: ${ ({ theme: { typo: { weights } } }) => weights.light };
```

Comment:
Why do you assign a font weight value to the `color` attribute?

Reply:
@Tomcatus This is not somethign i did, i simply took the code that was there at home and move it here so no idea.